### PR TITLE
Feature/stackn 246

### PR DESCRIPTION
**Pull request by stefanhellander**
## Status

- [ ] Ready
- [ ] Draft
- [ ] Hold

## Description

Describe your changes here to communicate to the maintainers why we should accept this pull request. 
If it fixes a bug or resolves a feature request, be sure to include a link to that issue.

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [x] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

**Review by dstoyanova — COMMENTED** [review]
The changes in this branch break the `build-jupyter-stackn.yaml` GitHub action since it cannot find `stackn/stackn/cli/Dockerfile` anymore. Maybe we should fix this before merging?

File: cli/stackn/delete_code [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
```
[the remark this answers is not in the document]
Comment (stefanhellander) [reply cli/stackn/delete_code:1]: No, will delete.

File: components/studio/projects/templates/project.html [hunk 2]
```diff
@@ -163,10 +163,7 @@ <h5 class="card-title mb-0">{{ objs.title }}</h5>
         </div>
     </div>
     {% endfor %}
-</div>
-
-<div class="row">
-    <div class="col-6 d-flex" style="padding-bottom: 24px;">
+    <div class="col-6 d-flex">
```
[the remark this answers is not in the document]
Comment (stefanhellander) [reply components/studio/projects/templates/project.html:166]: Hmm, I can't find the missing </div>...

File: cli/setup.py [hunk 3]
```diff
@@ -1,10 +1,9 @@
 from setuptools import setup, find_packages
-from scaleout import version
 
 
 setup(
     name='scaleout-cli',
-    version=version.__version__,
+    version='0.1.0',
```
[the remark this answers is not in the document]
Comment (stefanhellander) [reply cli/setup.py]: You are right, it does break stackn --version. Fixed now.

File: components/studio/projects/templates/project.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (dstoyanova) [reply components/studio/projects/templates/project.html:166]: It's between line 222 and line 223 according to my indentation tools :)

[the remark this answers is not in the document]
Comment (stefanhellander) [reply components/studio/projects/templates/project.html:166]: I can't find any problems with missing divs. I ran it through an html validator, and found a bunch of other small issues though that I fixed. :) Currently the following remain (I will leave those for now):

Error: The element button must not appear as a descendant of the a element.

From line 372, column 25; to line 372, column 66

          <button class="btn btn-outline-secondary">New</b

Error: The element button must not appear as a descendant of the a element.

From line 514, column 25; to line 514, column 66

          <button class="btn btn-outline-secondary">New</b

Error: The element button must not appear as a descendant of the a element.

From line 544, column 25; to line 544, column 66

          <button class="btn btn-outline-secondary">New</b

Error: The element button must not appear as a descendant of the a element.

From line 632, column 25; to line 632, column 66

          <button class="btn btn-outline-secondary">New</b

Error: Bad value https://torch-serve-aml-dbm-bd83.studio.local.stackn.dev/predictions/<model_name> for attribute href on element a: Illegal character in path segment: < is not allowed.

From line 683, column 59; to line 683, column 156

orting_1"><a href="https://torch-serve-aml-dbm-bd83.studio.local.stackn.dev/predictions/&lt;model_name&gt;">VGG se

Error: The element button must not appear as a descendant of the a element.

From line 750, column 25; to line 750, column 66

          <button class="btn btn-outline-secondary">New</b

Error: The element button must not appear as a descendant of the a element.

From line 780, column 25; to line 780, column 66

          <button class="btn btn-outline-secondary">New</b

Error: The element button must not appear as a descendant of the a element.

From line 1153, column 41; to line 1153, column 77

          <button class="btn mb-1 btn-twitter">↩     

Error: The element button must not appear as a descendant of the a element.

From line 1160, column 41; to line 1160, column 77

          <button class="btn mb-1 btn-youtube">↩     

Error: The element button must not appear as a descendant of the a element.

From line 1167, column 41; to line 1167, column 76

          <button class="btn mb-1 btn-github">↩

**Pull request merged**